pkg/sql/flowinfra/flow.go

```diff
@@ -123,13 +123,14 @@ type Flow interface {
 type FlowBase struct {
 	execinfra.FlowCtx
 
-	flowRegistry *FlowRegistry
+	// FlowRegistry is the registry that this flow is registered with.
+	FlowRegistry *FlowRegistry
 	// isVectorized indicates whether it is a vectorized flow.
 	isVectorized bool
-	// processors contains a subset of the processors in the flow - the ones that
+	// Processors contains a subset of the processors in the flow - the ones that
```

Comment:
Rather than exporting all of this stuff would it be better to call into FlowBase.Cleanup() from the various flow implementations which embed it and still have those implementations implement a Cleanup method?

Reply:
Yes, you're right. At first I thought that cleaning up of memory monitoring infrastructure in `vectorizedFlow`s forces us to actually copy-paste most of the logic, but rearranging the order of things slightly does the job. Done.